pages/components/cards.js

```diff
@@ -0,0 +1,201 @@
+        };
+      }
+      return post;
+    });
+    setPosts(updatedPosts);
+  };
+
+  // create Click on Image
+  const doubleClickHandler = (postId) => {
+    const updatedPosts = posts.map((post) => {
+      if (post.id === postId) {
+        return { ...post, likePhotos: true, likeStatus: true };
+      }
+      return post;
+    });
+    setPosts(updatedPosts);
+
+    setTimeout(() => {
+      const updatedPosts = posts.map((post) => {
+        if (post.id === postId) {
+          return { ...post, likePhotos: false, likeStatus: true };
+        }
+        return post;
+      });
+      setPosts(updatedPosts);
+    }, 1000);
+  };
+
+  return (
+    <Card>
+      {posts.map((post) => (
+        <div key={post.id}>
+          <PhotoContainer onClick={() => doubleClickHandler(post.id)}>
+            <Photo src={post.Photo} width={100} height={100} alt={post.title} />
+            {post.likePhotos && (
+              <LikeIconImage src="/assets/images/icons/like_fill.png" />
+            )}
+          </PhotoContainer>
+
+          <p>{post.content}</p>
+          <div>
+            <IconContainer>
+              <PhotoIcon
+                src="/assets/images/icons/comment.png"
+                width={30}
+                height={30}
+                alt="comment"
+                onClick={() => handleCommentClick(post.id)}
+              />
+              <PhotoIcon
+                src={
+                  post.likeStatus
+                    ? "/assets/images/icons/like_fill.png"
+                    : "/assets/images/icons/like.png"
+                }
+                width={30}
+                height={30}
+                alt="like"
+                onClick={() => handleLikeClick(post.id)}
+              />
+            </IconContainer>
+            {post.isCommentVisible && (
+              <form onSubmit={(event) => handleCommentSubmit(event, post.id)}>
+                <textarea
+                  placeholder="Enter your comment"
+                  value={post.comment}
+                  onChange={(event) => handleCommentUpdate(event, post.id)}
+                />
+                <button type="submit">Submit</button>
+              </form>
+            )}
+            {post.comments.map((submittedComment, index) => (
+              <CommentText key={index}>{submittedComment}</CommentText>
+            ))}
+          </div>
+          <br />
+          <hr />
+        </div>
+      ))}
+    </Card>
+  );
+}
+
+const Card = styled.div`
+  margin-bottom: 10px;
+`;
+
+const PhotoContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 100%;
+  position: relative;
+  overflow: hidden;
+`;
+
+const Photo = styled.img`
+  position: absolute;
+  top: 60px;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+`;
+
+const IconContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px 0 50px;
+`;
+
+const PhotoIcon = styled.img``;
```

Comment:
```suggestion
```

Reply:
it is necessary, otherwise styled components will give me a feh